.rubocop.yml

```diff
@@ -173,6 +182,9 @@ Style/WordArray:
   Exclude:
```

Comment:
Can we also change the WordArray to enforce brackets?
```
Style/WordArray:
  EnforcedStyle: brackets
```

This will need to run `rubocop -a` to change the code.